android/app/src/main/kotlin/net/mullvad/mullvadvpn/usecase/customlists/FilterCustomListsRelayItemUseCase.kt

```diff
@@ -33,7 +34,7 @@ class FilterCustomListsRelayItemUseCase(
                 providers = selectedProviders,
                 daita =
                     shouldFilterByDaita(
-                        isDaitaEnabled = settings?.isDaitaEnabled() == true,
+                        isDaitaEnabled = settings?.daitaAndDirectOnly() == true,
```

Comment:
Should this be more explicit? `isDaitaEnabled` is seemingly from this logic specifically `isDirectOnlyDaitaEnabled`?

Reply:
Changed